yarn.lock

```diff
@@ -8729,6 +8780,17 @@ readable-stream@4.4.0:
     events "^3.3.0"
     process "^0.11.10"
 
+readable-stream@4.4.2:
```

Comment:
note: looks like we brought in another version of `readable-stream` because `@tbd54566975/dwn-sql-store@0.0.3` isn't permissive wrt minor or patch version

Reply:
I can make a patch over there to fix this up.